evap/static/ts/src/sisyphus.ts

```diff
@@ -0,0 +1,312 @@
+/**
+ * Plugin developed to save html forms data to LocalStorage to restore them after browser crashes, tabs closings
+ * and other disasters.
+ *
+ * @author Alexander Kaupanin <kaupanin@gmail.com>
+ * @author Jonathan Weth <dev@jonathanweth.de>
+ */
+
+interface SisyphusOptions {
```

Comment:
A lot of the options are not needed, like `timeout` or `onBeforeRestore`. I think we can remove them